From 1741b1702e34f426c4e7b28f1b9a207ffeb4c7ac Mon Sep 17 00:00:00 2001
From: Loys Ollivier <loys@fb.com>
Date: Fri, 1 Mar 2019 17:01:33 +0100
Subject: [PATCH] pumpkin: add static alsa driver for audio sink

The jack output is not detected correctly by PulseAudio on the pumpkin
board.
Add a static driver that defines this sink in pulseaudio configuration.
Make this output the default output for pulseaudio.

Signed-off-by: Loys Ollivier <lollivier@baylibre.com>
---
 src/daemon/default.pa.in | 1 +
 src/pulse/client.conf.in | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index 14b6a6f969e4..a47486f0cf2c 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -38,6 +38,7 @@ load-module module-switch-on-port-available
 ### use module-udev-detect -- see below -- for doing this automatically)
 ifelse(@HAVE_ALSA@, 1, [dnl
 #load-module module-alsa-sink
+load-module module-alsa-sink device=hw:0,4
 #load-module module-alsa-source device=hw:1,0
 ])dnl
 ifelse(@HAVE_OSS_OUTPUT@, 1, [dnl
diff --git a/src/pulse/client.conf.in b/src/pulse/client.conf.in
index 69830efd2451..ee094ca8dad0 100644
--- a/src/pulse/client.conf.in
+++ b/src/pulse/client.conf.in
@@ -17,7 +17,7 @@
 ## more information. Default values are commented out.  Use either ; or # for
 ## commenting.
 
-; default-sink =
+default-sink = alsa_output.hw_0_4
 ; default-source =
 ; default-server =
 ; default-dbus-server =
-- 
2.7.4

